auth/gcloud/aio/auth/session.py

```diff
@@ -167,6 +175,7 @@ async def get(  # type: ignore[override]
             timeout: Timeout = 10,
             params: Optional[Mapping[str, Union[int, str]]] = None,
             stream: Optional[bool] = None,
+            allow_redirects: bool = False,
```

Comment:
Changing the default for GETs (afaik, this defaults to true in both aiohttp and requests) might cause some breaking changes which could be hard to debug or test for. Thoughts on having this be truthy by default?

Looks like you only need to disable redirects for HEADs, do we need/want to expose this at all here?

Reply:
That's a good point and thanks for catching that, for some reason when I read the documentation I thought it stated that `GET`s have `allow_redirects=False` by default, but that's only for `HEAD`.

Agreed, I'm okay removing `allow_redirects` for `GET` since we don't have any use case for it so KISS applies here.